cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagement.scala

```diff
@@ -124,24 +132,52 @@ object ClusterHttpManagementRoutes extends ClusterHttpManagementHelper {
       }
     }
 
+  private def routeGetShardInfo(cluster: Cluster, shardRegionName: String) =
+    get {
+      extractExecutionContext { implicit executor =>
+        complete {
+          implicit val timeout = Timeout(5.seconds)
+          try {
+            ClusterSharding(cluster.system)
+              .shardRegion(shardRegionName)
+              .ask(ShardRegion.GetShardRegionStats)
+              .mapTo[ShardRegion.ShardRegionStats]
+              .map { shardRegionStats =>
+                ShardDetails(shardRegionStats.stats.map(s => ShardRegionInfo(s._1, s._2)).toSeq)
+              }
+          } catch {
+            case _: IllegalArgumentException =>
+              StatusCodes.NotFound → ClusterHttpManagementMessage(s"Shard Region $shardRegionName is not started")
+          }
+        }
+      }
+    }
+
   /**
    * Creates an instance of [[akka.cluster.http.management.ClusterHttpManagementRoutes]] to manage the specified
    * [[akka.cluster.Cluster]] instance. This version does not provide Basic Authentication. It uses
    * the default path "members".
    */
-  def apply(cluster: Cluster): Route = apply(cluster, "members")
+  def apply(cluster: Cluster): Route = apply(cluster, "")
 
   /**
    * Creates an instance of [[akka.cluster.http.management.ClusterHttpManagementRoutes]] to manage the specified
    * [[akka.cluster.Cluster]] instance. This version does not provide Basic Authentication. It uses
    * the specified path `pathPrefixName`.
    */
   def apply(cluster: Cluster, pathPrefixName: String): Route =
-    pathPrefix(pathPrefixName) {
-      pathEndOrSingleSlash {
-        routeGetMembers(cluster) ~ routePostMembers(cluster)
+    rawPathPrefix(pathPrefixName) {
+      pathPrefix("members") {
```

Comment:
Hmm, this would break the API of any library user that specified a custom `pathPrefixName`, right?